### PR TITLE
Fixes #22677 Use proper api host name to allow TLS mutual auth

### PR DESCRIPTION
**Pull request by lphiri**
API requests must be made against "cert-api.access.redhat.com" and not "api.access.redhat.com".
The former is configured to accept RHSM manifest certificates for authentication, which is the expected production authentication method, with mutual tls verification enabled.
Reading the value of the API from a config file also allows this plugin to share configuration with the Insights plugin.

### CODE REVIEW

File: app/services/foreman_ansible/insights_plan_runner.rb [hunk 1]
```diff
@@ -27,10 +27,8 @@ def run_playbook
     # Fetches the playbook from the Red Hat Insights API
     def playbook
       resource = RestClient::Resource.new(
-        'https://api.access.redhat.com/r/insights/'\
-        "v3/maintenance/#{@plan_id}/playbook",
-        get_ssl_options_for_org(@organization, nil).
-        merge(:verify_ssl => OpenSSL::SSL::VERIFY_NONE)
+        "#{insights_api_host}/r/insights/v3/maintenance/#{@plan_id}/playbook",
```
Comment (dLobatog) [inline app/services/foreman_ansible/insights_plan_runner.rb]: @lphiri Where is the `insights_api_host` variable coming from?

Comment (lphiri) [reply app/services/foreman_ansible/insights_plan_runner.rb]: @dLobatog  its a method from the "RedhatAccess::Telemetry::LookUps" module - the same place you are getting " get_ssl_options_for_org".

**Comment by dLobatog** [comment]
[test]

**Comment by lphiri** [comment]
@dLobato The rubocop error (2:1) is nowhere near the code changes in this PR. Am I missing something?
I don't get any errors when I run ` rake foreman_ansible:rubocop`:
`[vagrant@centos7-dev-latest foreman]$ rake foreman_ansible:rubocop
/home/vagrant/foreman/lib/foreman.rb:8: warning: already initialized constant Foreman::UUID_REGEXP
/home/vagrant/foreman/lib/foreman.rb:8: warning: previous definition of UUID_REGEXP was here
Running RuboCop...
Inspecting 83 files
...................................................................................

83 files inspected, no offenses detected
`
I think the Firefox error is a script loading order issue (will not be an issue in production). I am looking into it.

**Pull request closed without merging**